# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/svg/crashtests/1572904.html
# Commit: 61a21b2c9e6e
# Full Hash: 61a21b2c9e6e9df619cf74f656d3989250939414
# Author: longsonr <longsonr@gmail.com>
# Date: 2019-08-20 09:38:33
# Description:
#   Bug 1572904 - refactor cycle collection code after DOMSVGAngle r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42453
# ==============================================================================

diff -r b85cb0cd1e49 -r 61a21b2c9e6e dom/svg/crashtests/1572904.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/svg/crashtests/1572904.html	Mon Aug 19 19:10:03 2019 +0000
@@ -0,0 +1,15 @@
+<script>
+function go() {
+  a.addEventListener("DOMSubtreeModified", go)
+  b.setAttribute("y", "")
+  SpecialPowers.gc()
+  SpecialPowers.forceCC()
+  try { c.viewBox.animVal[undefined] = ''} catch(e) { }
+  try { c.viewBox.animVal.x; } catch(e) { }
+}
+</script>
+<filter id="a">
+<svg id="" onload="go()">
+<path id="b">
+<view id="c" viewBox="1 9 0 1" />
+